src/send/EnterAmount.test.tsx

```diff
@@ -5,214 +5,163 @@ import { getNumberFormatSettings } from 'react-native-localize'
 import { Provider } from 'react-redux'
 import { SendEvents, TokenBottomSheetEvents } from 'src/analytics/Events'
 import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
-import { SendOrigin } from 'src/analytics/types'
-import { navigate } from 'src/navigator/NavigationService'
-import { Screens } from 'src/navigator/Screens'
-import { RecipientType } from 'src/recipients/recipient'
-import SendEnterAmount from 'src/send/SendEnterAmount'
-import { usePrepareSendTransactions } from 'src/send/usePrepareSendTransactions'
-import { getSupportedNetworkIdsForSend } from 'src/tokens/utils'
+import EnterAmount from 'src/send/EnterAmount'
```

Comment:
to minimise the diffs, i've removed the SendEnterAmount tests for now and done a rename here. these tests are mostly the same, but the EnterAmount component takes different props so most of the changes are related to those. the substance of the tests should be unchanged. i'll add a SendEnterAmount test file separately